src/syscheckd/src/create_db.c

```diff
@@ -1013,7 +1180,7 @@ fim_file_data *fim_get_data(const char *file, const directory_t *configuration,
     data->inode = statbuf->st_ino;
     data->dev = statbuf->st_dev;
     data->options = configuration->options;
-    data->last_event = time(NULL);
+    // data->last_event = time(NULL);
```

Comment:
Let's clean the commented code

Reply:
This is just to avoid a false positive when the entries are updated because this field is changed in every syncRow operation. This will be fixed in https://github.com/wazuh/wazuh/issues/12166.

I think we should leave this and uncomment this line when this issue is completed.